### PR TITLE
[feature](paimon/iceberg)add a docker that can directly pull up all the relevant environments of paimon/iceberg/doris

### PR DESCRIPTION
**Pull request by wuwenchi**
## Proposed changes

add a docker that can directly pull up all the relevant environments of paimon/iceberg/doris

### CODE REVIEW

File: samples/datalake/iceberg_and_paimon/README.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+[INFO] Submitting SQL update statement to the cluster...
+[INFO] SQL update statement has been successfully submitted to the cluster:
+Job ID: ff838b7b778a94396b332b0d93c8f7ac
+
+```
+
+After waiting for the task to be completed, we can view the modified data through doris.
+
+```sql
+mysql> refresh table customer;
```
Comment (morningman) [inline samples/datalake/iceberg_and_paimon/README.md:252]: No need to refresh

File: samples/datalake/iceberg_and_paimon/README.md [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+|                                                                                                                                                |
+|   HAS_COLO_PLAN_NODE: false                                                                                                                    |
+|                                                                                                                                                |
+|   STREAM DATA SINK                                                                                                                             |
+|     EXCHANGE ID: 01                                                                                                                            |
+|     UNPARTITIONED                                                                                                                              |
+|                                                                                                                                                |
+|   0:VPAIMON_SCAN_NODE(68)                                                                                                                      |
```
Comment (morningman) [inline samples/datalake/iceberg_and_paimon/README.md:291]: Only show this part

**Review by morningman — APPROVED** [review]
LGTM

**Comment by morningman** [comment]
run buildall

**Review by kaka11chen — APPROVED** [review]
LGTM

**Pull request merged**